src/_zkapauthorizer/schema.py

```diff
@@ -77,15 +74,15 @@ def run_schema_upgrades(upgrades, cursor):
         """
         CREATE TABLE [vouchers] (
             [number] text,
-            [created] text,                     -- An ISO8601 date+time string.
```

Comment:
Hm.  These text changes are all semantically null so they should be fine.  However, SQLite3 stores the schema as text in the database and as a result I'm vaguely afraid of changing the exact text of the old versions of the schema.

So I read https://sqlite.org/fileformat2.html#ffschema and I'm less afraid now.  The text is stored but of course SQLite3 has to parse it to actually do anything with it so non-semantic changes should make no difference to operation.  Further, SQLite3 already performs certain normalization to the text before storing it.  At first I thought it collapsed all whitespace in the statement down to a single space but on experimentation I see that it really only does this for whitespace between "create table" and the table name - not any subsequent whitespace.  Still, it seems the difference in the text should be harmless.

The moved comments are slightly different but it seems like they should likewise be harmless.

So, okay, no adjustment required here.